sormas-backend/src/test/java/de/symeda/sormas/backend/AbstractBeanTest.java

```diff
@@ -235,6 +239,16 @@ public void init() {
 		creator.createUser(null, null, null, "ad", "min", UserRole.ADMIN, UserRole.NATIONAL_USER);
 		when(MockProducer.getPrincipal().getName()).thenReturn("admin");
 
+		rdcf = creator.createRDCF("Region 1", "District 1", "Community 1", "Facility 1", "Point of entry 1");
```

Comment:
This is super class of a lot of other classes and none of them needs these variables except PersonFacadeEjbTest. Please override the init method in PersonFacadeEjbTest and move these variables there